utils/src/main/java/com/cloud/utils/backoff/impl/ConstantTimeBackoff.java

```diff
@@ -44,7 +41,7 @@
 public class ConstantTimeBackoff extends AdapterBase implements BackoffAlgorithm, ConstantTimeBackoffMBean {
     long _time;
     private final Map<String, Thread> _asleep = new ConcurrentHashMap<String, Thread>();
-    private final static Log LOG = LogFactory.getLog(ConstantTimeBackoff.class);
+    private static final Logger s_logger = Logger.getLogger(ConstantTimeBackoff.class.getName());
```

Comment:
name convention for private statics is all capitals so LOG is a better name.

Reply:
changed it to capitals but used S_LOGGER since this is the name we are using everywhere.

Reply:
no, please, the s_ is an old convention. LOGGER is ok, but LOG as well. I prefer the latter, but as long as we standardise on something I'm fine with it.